packages/utils/src/updateFTMetadata.ts

```diff
@@ -0,0 +1,32 @@
+import { ethers } from 'ethers'
+import { getMetadata } from '@near-eth/nep141-erc20/dist/natural-erc20'
```

Comment:
I think we should try to avoid making connector packages a dependency of @near-eth/utils to avoid the circular import.
For flexibility, the api of `updateFungibleTokenMetadata` can take a provider, nearAccount and metadata values as argument so that it can be used independently of @near-eth/client.